.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/HannesWell/m2e-core-tests.git
```

Comment:
@mihnita this way we can test changes in the m2e-core-tests submodule before merging them.
It is a bit more complicated and requires at least two verification-builds (if one wants to always have green builds), but it works.
Of course this has to be revered before this PR is submitted.

Reply:
I know. That part I did, and worked to test / fix things.

But I was unable to make the changes in `m2e-core-tests` show up in the `m2e-core` PR.
When I changed stuff and pushed it went in mihnita/m2e-core-tests.git, and I was able to make a PR from there.
But the PR from `m2e-core` ignored the changes in  `m2e-core-tests`

Reply:
> But the PR from `m2e-core` ignored the changes in `m2e-core-tests`

Changes of submodules are not shown in the container git, you only see that the ref-point to m2e-core-tests submodule has changed and you can only stage and commit that ref-pointer change. The staging view then looks like this:
![grafik](https://user-images.githubusercontent.com/44067969/185311750-21f79da8-2f7d-41ce-8448-940d88377f16.png)

But if you did that, maybe GH and Jenkins ignore changes in the .gitmodules of non-committers when checking out the submodules.

Reply:
For me it has worked to just edit the `.gitmodules` file and never changed the ref pointer....